From 4eeaf26a3c0d4a6dfa2db73becbc6a24d720c5cf Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Tue, 27 Aug 2024 18:41:56 +0000
Subject: [PATCH] desktop: Add back to AudioVideo category

The specification says that desktop entries that specify Audio as main category must include AudioVideo as well.
---
 podcasts-gtk/resources/org.gnome.Podcasts.desktop.in.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/podcasts-gtk/resources/org.gnome.Podcasts.desktop.in.in b/podcasts-gtk/resources/org.gnome.Podcasts.desktop.in.in
index e779dde48..39438ee91 100644
--- a/podcasts-gtk/resources/org.gnome.Podcasts.desktop.in.in
+++ b/podcasts-gtk/resources/org.gnome.Podcasts.desktop.in.in
@@ -7,7 +7,7 @@ Exec=gnome-podcasts
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;GTK;Audio;
+Categories=GNOME;GTK;AudioVideo;Audio;
 # Translators: Search terms to find this application. Do NOT translate or localize the semicolons! The list MUST also end with a semicolon!
 Keywords=Podcast;RSS;
 DBusActivatable=true
-- 
GitLab

